src/config.cpp

```diff
@@ -762,12 +762,18 @@ void list_string_f(std::unordered_map<std::string, std::string> &vars, const std
     input.emplace_back(begin, pos);
   }
 }
+
 void list_prep_cmd_f(std::unordered_map<std::string, std::string> &vars, const std::string &name, std::vector<prep_cmd_t> &input) {
   std::string string;
   string_f(vars, name, string);
 
   std::stringstream jsonStream;
 
+  // check if string is empty, i.e. when the value doesn't exist in the config file
+  if(string.empty()) {
+    string = "[]";
```

Comment:
This should work just fine. I think you could alternatively just `return` here and avoid the parsing altogether, but the end result is the same.